service/controller/controllercontext/context.go

```diff
@@ -11,7 +11,6 @@ type key string
 const contextKey key = "controller"
 
 type Context struct {
```

Comment:
The whole package should be deleted now when it is not used for anything anymore.